README.md

```diff
@@ -72,9 +72,13 @@ The Ditto Data Browser allows you to view all your collections, documents within
  <img src="/Img/collections.png" alt="Collections Image" width="300">  
 
  <img src="/Img/document.png" alt="Document Image" width="300">  
+ 
+**Standalone App**
+If you are using the Data Browser as a standalone app then there is a button `Start Subscriptions` you must press it in order to start syncing data.
```

Comment:
This: "...you must press it in order to start syncing data." should probably read: "...you must press in order to start syncing data.", grammatically, making the whole sentence:
`If you are using the Data Browser as a standalone app then there is a button, Start Subscriptions, you must press in order to start syncing data.`